src/main/java/net/neoforged/neoforge/common/conditions/IConditionBuilder.java

```diff
@@ -44,27 +44,11 @@ default ICondition tagEmpty(TagKey<Item> tag) {
         return new TagEmptyCondition(tag.location());
     }
 
-    default ICondition isFeatureEnabled(FeatureFlagSet requiredFeatures) {
-        return FlagCondition.isEnabled(requiredFeatures);
+    default ICondition featureFlags(FeatureFlagSet requiredFeatures) {
+        return FlagCondition.of(requiredFeatures);
     }
 
-    default ICondition isFeatureEnabled(FeatureFlag requiredFlag) {
-        return FlagCondition.isEnabled(requiredFlag);
-    }
-
-    default ICondition isFeatureEnabled(FeatureFlag requiredFlag, FeatureFlag... requiredFlags) {
-        return FlagCondition.isEnabled(requiredFlag, requiredFlags);
-    }
-
-    default ICondition isFeatureDisabled(FeatureFlagSet requiredFeatures) {
-        return FlagCondition.isDisabled(requiredFeatures);
-    }
-
-    default ICondition isFeatureDisabled(FeatureFlag requiredFlag) {
-        return FlagCondition.isDisabled(requiredFlag);
-    }
-
-    default ICondition isFeatureDisabled(FeatureFlag requiredFlag, FeatureFlag... requiredFlags) {
-        return FlagCondition.isDisabled(requiredFlag, requiredFlags);
+    default ICondition isFeatureEnabled(FeatureFlag... requiredFlags) {
```

Comment:
shouldnt this be renamed also, to match the above which was renamed?
```suggestion
    default ICondition featureFlags(FeatureFlag... requiredFlags) {
```